registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -181,6 +181,7 @@
     "sha256:931442dd6894a6f94177a7c36f93de5cb76986f084b2553f14dd0e53e9cf4727": ["v1.8.1"]
     "sha256:c1f8e1681c5d0890cb520101e0063d516110fd0c943240ebe0f049657738863b": ["v1.8.10"]
     "sha256:f93d4d3979d276529412bd427767fc022fdba521121fde05018d8e0add24b9b7": ["v1.8.11"]
+    "sha256:fbacf2c97b21131d2d694146814961bf97a606633d139bf3837676141de8c96a": ["v1.8.12"]
```

Comment:
`fbacf2c97b21` matches the SHA for cluster-api-controller v1.8.12 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:fbacf2c97b21131d2d694146814961bf97a606633d139bf3837676141de8c96a/details). 👍🏻